src/amo/containers/Home.js

```diff
@@ -17,49 +34,25 @@ export class HomePageBase extends React.Component {
       <div className="HomePage">
         <h2 className="HomePage-subheading">{i18n.gettext('What do you want Firefox to do?')}</h2>
         <ul className="HomePage-category-list">
-          <li className="HomePage-category-li HomePage-block-ads">
-            <Link to="#block-ads"><span>{i18n.gettext('Block ads')}</span></Link>
-          </li>
-          <li className="HomePage-category-li HomePage-screenshot">
-            <Link to="#screenshot"><span>{i18n.gettext('Screenshot')}</span></Link>
-          </li>
-          <li className="HomePage-category-li HomePage-save-stuff">
-            <Link to="#save-stuff"><span>{i18n.gettext('Save stuff')}</span></Link>
-          </li>
-          <li className="HomePage-category-li HomePage-shop-online">
-            <Link to="#shop-online"><span>{i18n.gettext('Shop online')}</span></Link>
-          </li>
-          <li className="HomePage-category-li HomePage-be-social">
-            <Link to="#share-stuff"><span>{i18n.gettext('Be social')}</span></Link>
-          </li>
-          <li className="HomePage-category-li HomePage-share-stuff">
-            <Link to="#share-stuff"><span>{i18n.gettext('Share stuff')}</span></Link>
-          </li>
+          <ExtensionLink name="block-ads" slug="security-privacy" text={i18n.gettext('Block ads')} />
```

Comment:
I was torn on this. I went with `text` since it makes it a little shorter but I'll switch it to `children`.